--- roadfighter-1.0.1269.orig/build/linux/Makefile	2008-03-19 21:58:51.000000000 +0100
+++ roadfighter-1.0.1269/build/linux/Makefile	2009-06-22 17:39:28.000000000 +0200
@@ -37,14 +38,14 @@
 	$(SRC)/main.o
 
 CC = g++
-CFLAGS = -g3 -O3 `sdl-config --cflags` -I/usr/X11R6/include
-LDFLAGS = `sdl-config --libs` -L/usr/X11R6/lib/ -lSDL_image -lSDL_mixer -lSDL_sound -lSDL_ttf
+CFLAGS += `sdl-config --cflags` -I/usr/X11R6/include
+LIBS = `sdl-config --libs` -L/usr/X11R6/lib/ -lSDL_image -lSDL_mixer -lSDL_sound -lSDL_ttf
+LDFLAGS += -L/usr/X11R6/lib/
 RM = rm -f
 CP = cp -r
 MD = mkdir -p
 ECHO = echo
 CHMOD = chmod
-STRIP = strip
 
 all: $(EXE)
 
@@ -52,8 +53,7 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 $(EXE): $(OBJS)
-	$(CC) $(LDFLAGS) $^ -o $@
-	@$(STRIP) $@
+	$(CC) $(LDFLAGS) $^ -o $@ $(LIBS)
 	@$(ECHO) " o If there are no errors, the game compiled succesfully"
 
 clean:
